environments/openshift/manifests/observatorium-template.yaml

```diff
@@ -1190,7 +1197,7 @@ parameters:
 - name: NAMESPACE
   value: telemeter
 - name: THANOS_IMAGE
-  value: improbable/thanos
+  value: quay.io/quay/thanos
```

Comment:
Just for your information, we override these variables before we deploy them. That's why they are a bit outdated.

Reply:
And it should be `quay.io/thanos-io/thanos`.